src/nxdoc/nuxeo-server/administration/database-configuration/documentdb.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Amazon DocumentDB
+labels:
+    - database
+    - documentdb
+tree_item_index: 700
+
+---
+`DocumentDB` is compatible with `MongoDB`, therefore, it can be used by `Nuxeo` like a `MongoDB` backend,
+but there are limitations.
+
+## Limitations
+
+- fulltext search at repository level using `DocumentDB` is not supported due to `DocumentDB` limitations. You need to rely on `Elasticsearch` or `OpenSearch`. This requires to disable repository fulltext search. See the [configuration]({{page space='nxdoc' page='documentdb#configuration'}}) section.
+- retryable writes are not supported. You need to disable this. See the [configuration]({{page space='nxdoc' page='documentdb#configuration'}}) section.
+- the `DocumentDB` instance type chosen affects the number of open cursor at any given time.
+
+Note there are also other differences that should not affect Nuxeo. Please refer to `DocumentDB`.
```

Comment:
would be interesting to point to the DocumentDB page explaining the MongodB support limitations

Reply:
Added back the 2 major links.